src/lwc/contactTile/contactTile.html

```diff
@@ -0,0 +1,19 @@
+<template>
```

Comment:
Where is this ContactTile will show on UI? Is this part of another component? In the JS part, the variable contact has a decorator as API so I assume that will be loaded from another component.